src/modules/handleInputOutput.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const readline = require('readline');
+
+const handleInputOutput = () => {
+  return new Promise(resolve => {
+    const message = 'Insert 4  different digits: ';
+
+    const terminal = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+
+    // eslint-disable-next-line no-console
+    console.log(message);
```

Comment:
You can avoid using eslint exceptions by writing `global.console.log(message);`

Reply:
thanks for that